play/src/front/Components/MapEditor/PropertyEditor/ListAddPropertyButton.svelte

```diff
@@ -0,0 +1,222 @@
+<script lang="ts">
```

Comment:
This `ListAddPropertyButton` seems weird to me. What is the interest of adding this component instead of keeping the `AddPropertyButton` directly in the `EntityPropertiesEditor.svelte` file?
Is this because you don't want to repeat the `AddPropertyButton` props over `EntityPropertiesEditor.svelte` and `AreaPropertiesEditor.svelte`?

If so, maybe we can find a cleaner way to do this?

What about creating one component per button?

Something like:

**FocusableButton.svelte**
```
<script lang="ts">
    import { createEventDispatcher } from "svelte";
    import {
        AreaDataPropertiesKeys,
        EntityDataPropertiesKeys,
        OpenWebsiteTypePropertiesKeys,
    } from "@workadventure/map-editor";
    import LL from "../../../../i18n/i18n-svelte";
    const dispatch = createEventDispatcher();
</script>

<AddPropertyButton
    headerText={$LL.mapEditor.properties.focusableProperties.label()}
    descriptionText={$LL.mapEditor.properties.focusableProperties.description()}
    img={"resources/icons/icon_focus.png"}
    style="z-index: 16;"
    on:click={(event) => {
        dispatch("click", event);
    }}
/>
```

This way, we avoid the weird component containing plenty of "if".

Reply:
This component has been created to reuse this component easelly in the app without import all of dependencies regarding "AddPropertyButton"